pageserver/src/tenant/gc_block.rs

```diff
@@ -174,22 +201,23 @@ impl std::fmt::Display for BlockingReasons {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(
             f,
-            "{} timelines block for {:?}",
-            self.timelines, self.reasons
+            "tenant_post_attached_single_wait: {}; {} timelines block for {:?}",
```

Comment:
assertion has comma, this has semi-colon.

the string is a crutch anyways I'll try to get rid of in near future, wouldn't sweat too much on it.